server/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EngineJobExecutionActorSpecUtil.scala

```diff
@@ -118,6 +118,12 @@ private[ejea] trait HasJobFailureResponses { self: EngineJobExecutionActorSpec =
 private[ejea] trait HasCopyFailureResponses { self: EngineJobExecutionActorSpec =>
   val copyFailureReason =
     new Exception("Deliberate failure for test case: failed to copy cache outputs!") with NoStackTrace
+
   // Need to delay making the response because job descriptors come from the per-test "helper", which is null outside tests!
-  def failedToCopyResponse(attemptNumber: Int) = CopyingOutputsFailedResponse(helper.jobDescriptorKey, attemptNumber, copyFailureReason)
+  def loggableFailedToCopyResponse(attemptNumber: Int) = CopyingOutputsFailedResponse(helper.jobDescriptorKey, attemptNumber, LoggableCacheCopyError(copyFailureReason))
+  def metricableFailedToCopyResponse(attemptNumber: Int) = CopyingOutputsFailedResponse(helper.jobDescriptorKey, attemptNumber, MetricableCacheCopyError(HasCopyFailureResponses.metricContext))
+}
+
+object HasCopyFailureResponses {
+  val metricContext = "metriccontext"
```

Comment:
is this the intended value?

Reply:
it's just a test, so I didn't put _too_ much thought into it's value

Reply:
Alternative suggestions welcome, of course...